internal/operators/lvm/lvm_operator_test.go

```diff
@@ -108,6 +108,14 @@ var _ = Describe("Lvm Operator", func() {
 				&common.Cluster{Cluster: models.Cluster{HighAvailabilityMode: &fullHaMode, Hosts: []*models.Host{hostWithSufficientResources, hostWithSufficientResources}, OpenshiftVersion: operator.config.LvmMinMultiNodeSupportVersion}},
 				api.ValidationResult{Status: api.Success, ValidationId: operator.GetHostValidationID()},
 			),
+			table.Entry("High Availability Mode Full with pre-release version",
+				&common.Cluster{Cluster: models.Cluster{HighAvailabilityMode: &fullHaMode, Hosts: []*models.Host{hostWithSufficientResources, hostWithSufficientResources}, OpenshiftVersion: "4.15.0-rc0"}},
+				api.ValidationResult{Status: api.Success, ValidationId: operator.GetHostValidationID()},
+			),
+			table.Entry("High Availability Mode Full with higher than LvmMinMultiNodeSupportVersion",
```

Comment:
Can we add SNO with openshift version > LvmMinMultiNodeSupportVersion?

Reply:
already there - the naming is a big confusing   
```go
noneHaMode := models.ClusterHighAvailabilityModeNone
```
https://github.com/openshift/assisted-service/blob/a2095745415dd0d8b680808b74924eb3bccd0b47/internal/operators/lvm/lvm_operator_test.go#L123-L130

Reply:
yep sorry I've missed it :sweat: